fn invalid_output() -> &str { 
    "foo"
}

// fix the error and print invalid_output()
fn main() {}